README.md

```diff
@@ -158,10 +158,130 @@ This will determine any missing `ansible-core` tags and create them in `ansible-
+There are various other changes that should occur around the same time that the new stable branch is cut.
+
+### Creating stable branches
+
+Someone with maintainer access must create the new branch.
+You can do so as follows:
+
+```bash
+# Make sure your checkout is up to date.
+git fetch upstream
+
+# Create a new stable branch against the devel branch.
+git checkout -b stable-2.18 upstream/devel
+
+# Push the new stable branch to the repository.
+git push upstream stable-2.18
+```
+
+After the new stable branch is created, the following changes should be committed as pull requests to the new stable branch:
+
+* Update the core branch in the `docs/ansible-core-branch.txt` file.
+* Remove devel-only tooling.
+* Update Python versions in the support matrix.
+
+### Updating the core branch
+
+The script that grafts portions of the core repository uses the `docs/ansible-core-branch.txt` file to specify which branch to clone.
+When a new stable branch is created, modify the file so that it specifies the correct version.
+
+```bash
+sed -i 's/devel/stable-2.18/g' docs/ansible-core-branch.txt
+```
+
+### Remove devel-only tooling
+
+There are some scripts and other tooling artefacts that should be on the `devel` branch only.
+After creating a new stable branch, remove the appropriate files and references.
+
+```bash
+# Remove the following workflow files, the tagger script, and tagger requirements.
+git rm -r .github/workflows/pip-compile-dev.yml .github/workflows/pip-compile-docs.yml .github/workflows/reusable-pip-compile.yml .github/workflows/tag.yml hacking/tagger tests/tag.*
+```
+
+Next, remove references to the tagger dependencies as follows:
+
+1. Remove the reference from the typing input file.
+
+   ```bash
+   sed -i '/-r tag.in/d' tests/typing.in
+   ```
+
+2. Clean up the typing lockfile.
+
+   ```bash
+   nox -s pip-compile -- --no-upgrade
+   ```
+
+Finally, open `noxfile.py` and remove `"hacking/tagger/tag.py",` from the `LINT_FILES` tuple, for example:
+
+#### Previous lint files tuple
+
+```python
+LINT_FILES: tuple[str, ...] = (
+    "hacking/pr_labeler/pr_labeler",
+    "hacking/tagger/tag.py",
+    "noxfile.py",
+    *iglob("docs/bin/*.py"),
+)
+```
+
+#### Updated lint files tuple
+
+```python
+
+LINT_FILES: tuple[str, ...] = (
+    "hacking/pr_labeler/pr_labeler",
+    "noxfile.py",
+    *iglob("docs/bin/*.py"),
+)
+```
+
+### Update Python versions in the support matrix
+
+The minimum supported Python version changes with each Ansible core version.
+This requires an update to the support matrix documentation after a new stable branch is created to reflect the appropriate Control Node Python versions.
+
+Uncomment the new stable version from the `ansible-core support matrix` section in the `docs/docsite/rst/reference_appendices/release_and_maintenance.rst` file.
+Submit a PR with the changes and request a core team review.
+
```

Comment:
Everything else here is technical changes to get the new branch to work. This is a docs change. Do we really want this here in the branching process? It's already part of the documentation checklist issues (tho we should document that maybe also in that newe maintainers.md file in a subsequent pr).

Reply:
Nice catch @samccann I was thinking the same thing about the maintainers guide. I've decided to add `maintainers.md` so we can leave this detail in place for now. In a follow up, we can move this part to a new section about the release checklist. How does that sound?